modules.d/90dmsquash-live/dmsquash-live-root.sh

```diff
@@ -390,12 +442,26 @@ if [ -n "$overlayfs" ]; then
         info "Resetting the OverlayFS overlay directory."
         rm -r -- ${ovlfs}/* ${ovlfs}/.* >/dev/null 2>&1
     fi
+    if [ -n ${join} ]; then
+        mkdir -m 0755 /run/rootfsjoin
+    fi
     if [ -n "$readonly_overlay" ] && [ -h /run/overlayfs-r ]; then
-        ovlfs=lowerdir=/run/overlayfs-r:/run/rootfsbase
+        if [ -n ${join} ]; then
+            ovlfs=lowerdir=/run/overlayfs-r:/run/rootfsjoin:/run/rootfsbase
+        else
+            ovlfs=lowerdir=/run/overlayfs-r:/run/rootfsbase
+        fi
     else
-        ovlfs=lowerdir=/run/rootfsbase
+        if [ -n ${join} ]; then
+            ovlfs=lowerdir=/run/rootfsjoin:/run/rootfsbase
+        else
+            ovlfs=lowerdir=/run/rootfsbase
+        fi
     fi
     mount -r $FSIMG /run/rootfsbase
+    if [ -n ${join} ]; then
+        mount -r $FSIMG_JOIN /run/rootfsjoin
+    fi
     if [ -z "$DRACUT_SYSTEMD" ]; then
```

Comment:
In order to support DRACUT_SYSTEMD systems, the ovlfs variable in dmsquash-generator.sh must be similarly modified.